docs/index.rst

```diff
@@ -30,8 +30,8 @@ Pillow is the friendly PIL fork by `Alex Clark and Contributors <https://github.
    :target: https://pypi.org/project/Pillow/
    :alt: Number of PyPI downloads
 
-.. image:: https://coveralls.io/repos/python-pillow/Pillow/badge.svg?branch=master&service=github
-   :target: https://coveralls.io/github/python-pillow/Pillow?branch=master
+.. |coverage| image:: https://codecov.io/gh/python-pillow/Pillow/branch/master/graph/badge.svg
```

Comment:
```suggestion
.. image:: https://codecov.io/gh/python-pillow/Pillow/branch/master/graph/badge.svg
```

This is an RST substitution definition that is not used. I presume it exists because it was copied from README.